images/scripts/fedora.setup

```diff
@@ -172,7 +164,7 @@ rpm -e --verbose man-db
 # no need to install all dependencies there
 ALL_DEPS="$COCKPIT_DEPS"
 if [ "fedora-eln" != "$IMAGE" ]; then
-    ALL_DEPS="$COCKPIT_MACHINES_DEPS $COCKPIT_STORAGE_DEPS $COCKPIT_DEPS $IPA_CLIENT_PACKAGES $TEST_PACKAGES $ABRT_PACKAGES"
+    ALL_DEPS="$COCKPIT_MACHINES_DEPS $COCKPIT_STORAGE_DEPS $COCKPIT_DEPS $IPA_CLIENT_PACKAGES $TEST_PACKAGES"
```

Comment:
This is too much. We still want to keep the ABRT_PACKAGES on fedora-{39,40}, just not on any newer release (i.e. rawhide).

Reply:
OK, I think it should be better now